### PR TITLE
Add quickstarts and samples to README

### PR DESCRIPTION
**Pull request by brendanzagaeski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ This repo is designed as a resource for developers building applications that us
 Feel free to create an issue if you have a question, suggestion, or would like to report an issue to the Azure Cache team. We welcome feedback on the cache product itself, the contents of this repo, or anything else. 
 
 ### Table of contents: 
+
+- [Quickstarts](https://github.com/Azure-Samples/azure-cache-redis-samples)
+  - Small projects that demonstrate the essential steps to incorporate Azure Cache for Redis into an app.
+- [Samples](https://github.com/Azure-Samples/azure-cache-redis-samples)
+  - Sample projects that demonstrate additional usage patterns for Azure Cache for Redis.
```
Comment (brendanzagaeski) [inline README.md:12]: For the moment, I kept these entries brief. As this README evolves, one other option to consider might be to copy and paste the full list of individual quickstarts and samples from the azure-redis-cache-samples README.

File: README.md [hunk 2]
```diff
@@ -5,6 +5,11 @@ This repo is designed as a resource for developers building applications that us
 Feel free to create an issue if you have a question, suggestion, or would like to report an issue to the Azure Cache team. We welcome feedback on the cache product itself, the contents of this repo, or anything else. 
 
 ### Table of contents: 
+
+- [Quickstarts](https://github.com/Azure-Samples/azure-cache-redis-samples)
+  - Small projects that demonstrate the essential steps to incorporate Azure Cache for Redis into an app.
```
Comment (philon-msft) [inline README.md:10]: Let's put these links in a new section called "Other Resources" after the "Table of contents" section.

**Review by philon-msft — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (brendanzagaeski) [reply README.md:10]: Ooh, yeah. That makes sense. Let me move them.

**Review by philon-msft — APPROVED** [review]

**Comment by brendanzagaeski** [comment]
**Changes in the latest force-push:**

Move the links for the quickstarts and samples under an _Additional resources_ heading.

I made one small modification from "Other resources" to "Additional resources" since by chance I noticed the heading "Additional resources" in some of the Azure Cognitive Services docs, but I'm happy to switch that to "Other resources" if "Additional resources" doesn't look quite right. Thanks!

**Comment by brendanzagaeski** [comment]
**Changes in the latest force-push:**

Make one tiny adjustment to add a colon on "Additional resources:" so it's consistent with "Table of contents:"

**Pull request merged**